docs/release-process.md

```diff
@@ -5,15 +5,35 @@
 - run `bolt changeset` - select all packages affected by your change
 - Fill out a description of the change - this will end up in the changelog
 - If it was too hard to write everything in one line, go to the `.changeset/**/changes.md` file and flesh out the change. The entire markdown chunk here will be written to your changeset, so feel free to include code snippets or suggestions on how to upgrade if they are needed
-    - IF IT WAS __A BREAKING CHANGE__, you should definitely do this
+  - IF IT WAS **A BREAKING CHANGE**, you should definitely do this
 - It’s better to make multiple changesets, even for the same packages, to get good changelog entries, rather than trying to condense them and end up with useless changelogs
 
 ## How releases work
 
 We currently do manual releases. Our release process looks like:
 
-- Declare a release is being organised, block merging to master
-- Have a release manager run `build-releases version`
-- PR the changes from this script back into master - get a review of changelogs, and packages will be published
-- Once merged, re-pull master, manually run `build-releases publish`, push tags
-- Let everyone know it's safe to merge PRs again
\ No newline at end of file
+### Extra stuff on releasing
+
+We have had several incidents where we have run a release, having had tests pass, and run some local checks, and still broken things.
+
+The most common breakages are links and images. This is because of three major reasons:
+
+1. Running `brisk build && brisk export` produces different results than running `brisk dev`, which we usually use for testing
+2. Brisk's own documentation has not had images, so we haven't looked at it
```

Comment:
Add a reference to a github issue here so we know when it's safe to remove this from the docs